beamer/tests/agent/multiple_processors/test_multiple_event_processors.py

```diff
@@ -0,0 +1,224 @@
+import contextlib
+import json
+import os
+import signal
+import time
+from pathlib import Path
+from subprocess import DEVNULL, PIPE
+
+import ape
+import eth_account
+import psutil
+import yaml
+from eth_utils import to_canonical_address
+from yaml.loader import SafeLoader
+
+from beamer.agent.agent import Agent
+from beamer.agent.config import Config
+from beamer.agent.contracts import ContractInfo
+from beamer.agent.typing import BlockNumber, ChainId
+from beamer.agent.util import TokenChecker, make_web3, transact
+from beamer.tests.conftest import Contracts
+from beamer.tests.util import Sleeper
+
+SLAVE_TEST_PATH = Path(__file__).parent / "_test_slave.py"
+CONFIG_PATH = ape.project.local_project.path / ape.project.local_project.config_file_name
```

Comment:
Nice, this is much better, thanks for fixing it.
Both of those should be private, though.